pkg/database/database.go

```diff
@@ -71,6 +73,22 @@ func NewMasterKeyAuthorizer(ctx context.Context, log *logrus.Entry, token azcore
 	return cosmosdb.NewMasterKeyAuthorizer(getDatabaseKey(keys, log))
 }
 
+func NewTokenAuthorizer(ctx context.Context, log *logrus.Entry, cred azcore.TokenCredential, databaseAccountName string, scopes []string) (cosmosdb.Authorizer, error) {
+	scopes = append(scopes, fmt.Sprintf("https://%s.documents.azure.com/.default", databaseAccountName))
```

Comment:
This scope is incorrect for government cloud (and any non-Azure Public Cloud)

We should read the url from the env struct we use everywhere to be cloud aware.

https://learn.microsoft.com/en-us/azure/azure-government/compare-azure-government-global-azure

Reply:
Gotcha! .. sorry for that, I'll correct it.